app/src/main/java/com/github/ybecker/epforuml/AccountFragment.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.ybecker.epforuml
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [AccountFragment.newInstance] factory method to
+ * create an instance of this fragment.
```

Comment:
Maybe you could explain what is your fragment used for in more details in this comment. 
(This remark holds for all fragments)